pkg/virtctl/update/machine-type/convert-machine-type/controller.go

```diff
@@ -0,0 +1,162 @@
+package convertmachinetype
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	k8sv1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+	k6tv1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+)
+
+var ExitJob chan struct{}
```

Comment:
I think that this needs to go away, and instead get a stop channel in the controller's constructor.
Also bear in mind that only the tests actually initialize this channel.